test/shopify_api/middlewere/event_struct_coercer_test.exs

```diff
@@ -53,11 +75,10 @@ defmodule ShopifyAPI.Middleware.EventStructCoercerTest do
       |> build_pipeline
       |> EventStructCoercer.before_work()
 
-    assert ^pipe =
-             build_pipeline([
-               {:not_an_event},
-               %{type: "not and event"},
-               %Event{destination: "is_event", action: "action", token: %{}}
-             ])
+    assert [
+             {:not_an_event},
+             %{type: "not and event"},
```

Comment:
Is this a typo?

Reply:
It sure is. Spellcheck only gets me so far ...